doc/appendix-de-kasssichv/appendix-de-kasssichv.md

```diff
@@ -96,57 +96,168 @@ For Germany (DE) the country code is `0x4445`. Thus, the value for an unknown ft
+| `0x444500000000xxxx` | daily-closing<br /> TBD: close all open cbReceiptReference <-> Transaction-ID  <br />this works only on implicit flow. calling without the ReceiptCaseFlag 0xyyyy ends up in an exception. | 1.3- |
+| `0x444500000000xxxx` | monthly-closing<br /> TBD: close all open cbReceiptReference <-> Transaction-ID  <br />this works only on implicit flow. calling without the ReceiptCaseFlag 0xyyyy ends up in an exception. | 1.3- |
+| `0x444500000000xxxx` | yearly-closing<br /> TBD: close all open cbReceiptReference <-> Transaction-ID  <br />this works only on implicit flow. calling without the ReceiptCaseFlag 0xyyyy ends up in an exception. | 1.3- |
+| `0x444500000000xxxx` | b2b-invoice<br /> TBD: <br /> DSFinV-K: BON_TYP=Beleg, can be overwritten by ftReceiptCaseFlag  | 1.3- |
+| `0x444500000000xxxx` | b2c-invoice<br /> TBD: <br /> DSFinV-K: BON_TYP=Beleg, can be overwritten by ftReceiptCaseFlag  | 1.3- |
+| `0x444500000000xxxx` | info-invoice<br /> TBD: <br /> DSFinV-K: BON_TYP=AVRechnung | 1.3- |
+| `0x444500000000xxxx` | info-delivery-note<br /> TBD: <br /> DSFinV-K: BON_TYP=AVTransfer | 1.3- |
+| `0x444500000000xxxx` | info-order<br /> TBD: <br /> DSFinV-K: BON_TYP=AVBestellung | 1.3- |
+| `0x444500000000xxxx` | cash deposit / cash pay-in / cash pay-out / exchange<br /> TBD: <br /> DSFinV-K: BON_TYP=Beleg, can be overwritten by ftReceiptCaseFlag | 1.3- |
+| `0x444500000000xxxx` | material consumption<br /> TBD: <br /> DSFinV-K: BON_TYP=AVSachbezug | 1.3- |
+| `0x444500000000xxxx` | info-internal<br /> TBD: <br /> DSFinV-K: BON_TYP=AVSonstige | 1.3- |
+| `0x444500000000xxxx` | protocol<br /> TBD: <br /> DSFinV-K: BON_TYP=AVSonstige | 1.3- |
+| `0x444500000000xxxx` | foreign sales<br /> TBD: <br /> DSFinV-K: BON_TYP=AVSonstige | 1.3- |
+
+
+
+
+##### actions where a receiptcase has to be defined
+
+- register new terminal (TerminalID)
+- unregister a terminal (TerminalID)
+
+@startuml
+
+POS -> Queue : ReceiptRequest
+
+@enduml
+
+
 
 #### ftReceiptCaseFlag
 
 This table expands on the values provided in table 10 of chapter x.y.z on page p with values applicable to the German market.
 
 | Value | Description | Service-Version |
-|----------------------|
-| TBD | Implicit Transaction. No Start-Transaction call to ´Sign´ is required, it is done implicit. If the unique identifier set in property ´cbReceiptIdentification´ already started a transaction, this will throw an exception. | TBD |
+|---|---|---|
+| 0x0000000000010000 | out of service | 1.3- |
+| 0x0000000000020000 | training receipt<br /> DSFinV-K: overrides BON_TYP=AVTraining  | 1.3- |
+| 0x0000000000040000 | reverse/voided receipt<br /> DSFinV-K: overrides BON_TYP=AVBelegstorno  | 1.3- |
+| 0x0000000000080000 | handwritten receipt | 1.3- |
```

Comment:
this is the same behaviour as in the general part and valid from version 1.0, so ot shouldn't be mentioned

Reply:
there will be a special flow for german market. also the DSFinV-K behaviour is in addition to the default one